### DISCUSSION TITLE
[create-aptos-dapp] Error:"While processing dependency 'AptosFramework': Unable to find package manifest"

### DISCUSSION THREAD

**Discussion by NaDasai** [aptos-labs/aptos-developer-discussions#334] [Questions] [answered]
### Discord user ID

ibuidl

### Describe your question in detail.

Hello!
I have this error when I try to **compile** the create-aptos-dapp (all of them), the only thing I changed is the VITE_FA_CREATOR_ADDRESS in the .env :

>   "Error": "Move compilation failed: Unable to resolve packages for package 'fungible_asset_launchpad': While resolving dependency 'AptosFramework' in package 'fungible_asset_launchpad': While processing dependency 'AptosFramework': Unable to find package manifest for 'AptosFramework' at \"/Users/nadhir/.move/https___github_com_aptos-labs_aptos-core_git_main/aptos-move/framework/aptos-framework\""

Here is my Move.toml:

> [package]
name = "fungible_asset_launchpad"
version = "1.0.0"
authors = []

> [addresses]
> launchpad_addr = '_'
> initial_creator_addr = "_"
> 
> [dependencies]
> AptosFramework = { git = "https://github.com/aptos-labs/aptos-core.git", rev = "mainnet", subdir = "aptos-move/framework/aptos-framework"}
> 
> [dev-dependencies]

### Which operating system are you using?

macOS

### Which SDK or tool are you using? (if any)

Aptos CLI

**Comment by 0xmaayan** [comment]
Hi @NaDasai. 

1. Could you clarify how you compile the package? is it by the `npm run move:compile` command? if so, do you run it on the root folder of your project?
2. What do you mean by "the only thing I changed is the VITE_FA_CREATOR_ADDRESS in the .env"? have you changed the variable name? or just assigned a value?

**Reply by NaDasai** [reply]
Hi @0xmaayan 
1. Yes I used `npm run move:compile`. And yes in the root folder of my project (the folder containing all the project, the folder with the name of your project right?).
2. No, I just assigned a value to it:
`VITE_FA_CREATOR_ADDRESS=0x59a37cd9225f3e20429898c59a682418b332dd74122414a55327dd25bc91386e`

Looks like I can't reach the github repo.. I have errors in my .move file: `Unresolved Type`. Tried to compile to try to get the dependencies but I have tha message error.

**Reply by 0xmaayan** [reply, marked as answer]
Hi @NaDasai ! thanks for clarifying.

It looks like it is a network issue when trying to download the move dependencies. I'd suggest trying different networks (or vpns).

**Reply by NaDasai** [reply]
Hi @0xmaayan Thanks for your reply!
I tried with other networks but I still the same errors. Should I use a vpn (any suggestions, don't have one)?

Btw I have this message when I open the project, I select the Path suggested. Just wanted to make sure it's the right one (I mean the path exists but want to make sure it's the right one.
<img width="897" alt="path" src="https://github.com/user-attachments/assets/c8482b0b-c7bf-45ca-827e-250cd2cbfdd7">

**Reply by 0xmaayan** [reply]
@NaDasai yes it looks fine - you could validate it by running `which aptos` on your local terminal/cli

I suggest you try out some known VPNs in you area and see if one of them works.

**Reply by NaDasai** [reply]
Hey! I used the VPN but still have the same error.. (btw we are many to have the same error, so it's not only me)

**Reply by SiddharthManjul** [reply]
I am facing the same issue

**Reply by NaDasai** [reply]
@SiddharthManjul  @0xmaayan 
This a _workaround_ I found:
1/ Download the dependencies in your computer from Github.
2/ Then change the path in Move.toml file.
In my case:

> [dependencies]
AptosFramework = { local = "/Users/nad/Documents/Aptos/aptos-core-main/aptos-move/framework/aptos-framework" , rev="testnet" }

**Reply by NaDasai** [reply]
@0xmaayan Oh and looks like someone fixed the same issue by adding this path (Windows):
`C:\Program Files\Git\bin\git.exe `
(this could be a clue)

**Reply by SiddharthManjul** [reply]
I tried it already but did not include the complete path as it was there in subdir
